packages/datasource-customizer/test/decorators/operators-emulate/collection.test.ts

```diff
@@ -157,6 +157,14 @@ describe('OperatorsEmulateCollectionDecorator', () => {
       });
     });
 
+    describe('when given null handler value for replaceFieldOperator', () => {
```

Comment:
```suggestion
     test('replaceFieldOperator() should throw if no handler is given', () => {
```
If we want to stick to the previous tests hierarchy.